nginx/config/base.go

```diff
@@ -124,15 +124,14 @@ http {
   # Gzip Settings
   ##
 
-  gzip on;
-  gzip_disable "msie6";
-
+  # gzip on;
```

Comment:
Can we delete commented out code? Sort of confusing.

Reply:
I think some standard options are commented out because it would be easier to customize. Let me change that to `gzip off` instead?